pkg/storage/driver/driver.go

```diff
@@ -17,20 +17,9 @@ limitations under the License.
 package driver // import "k8s.io/helm/pkg/storage/driver"
 
 import (
-	"fmt"
-
 	rspb "k8s.io/helm/pkg/proto/hapi/release"
 )
 
-var (
-	// ErrReleaseNotFound indicates that a release is not found.
-	ErrReleaseNotFound = func(release string) error { return fmt.Errorf("release: %q not found", release) }
```

Comment:
can we do a `ErrReleaseNotFound = storageerrors.ErrReleaseNotFound`? That way we won't break other users relying on this API.

```
diff --git a/pkg/storage/driver/driver.go b/pkg/storage/driver/driver.go
index 7ad28770..017468d6 100644
--- a/pkg/storage/driver/driver.go
+++ b/pkg/storage/driver/driver.go
@@ -17,9 +17,20 @@ limitations under the License.
 package driver // import "k8s.io/helm/pkg/storage/driver"

 import (
+       storageerrors "k8s.io/helm/pkg/storage/errors"
+
        rspb "k8s.io/helm/pkg/proto/hapi/release"
 )

+var (
+       // ErrReleaseNotFound has been deprecated; please use storageerrors.ErrReleaseNotFound instead.
+       ErrReleaseNotFound = storageerrors.ErrReleaseNotFound
+       // ErrReleaseExists has been deprecated; please use storageerrors.ErrReleaseExists instead.
+       ErrReleaseExists = storageerrors.ErrReleaseExists
+       // ErrInvalidKey has been deprecated; please use storageerrors.ErrInvalidKey instead.
+       ErrInvalidKey = storageerrors.ErrInvalidKey
+)
+
 // Creator is the interface that wraps the Create method.
 //
 // Create stores the release or returns ErrReleaseExists
```